spark/spark-1.6/pom.xml

```diff
@@ -9,13 +9,12 @@
 		<relativePath>../../pom.xml</relativePath>
 	</parent>
 
-	<artifactId>hivemall-spark</artifactId>
-	<name>Hivemall on Spark</name>
+	<artifactId>hivemall-spark_2.11</artifactId>
```

Comment:
This means this jar supports spark-2.0 compiled with scala-2.11.
This naming rule is along with sprak jar.